core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.html

```diff
@@ -1,22 +1,22 @@
 <dot-dropdown-component
```

Comment:
All this needs do be a `p-menu` even the header with the name of the avatar

<img width="303" alt="image" src="https://user-images.githubusercontent.com/751424/226477590-94fb8852-6a77-462a-86e5-ac1caea1caf8.png">

In the #23678 I explain how you can add HTML to the menu items.

Reply:
Yeah I tried that, but that header uses the `dot-gravatar` component and `p-menu` just accept plain HTML it just ignore the `<dot-gravatar>`

Reply:
I realized that there is a gravatar service, I can get the URL to set the image